### ISSUE TITLE
fails with tramp

### ISSUE THREAD

**Issue by unhammer** [steckerhalter/emacs-fasd#11]
If I have opened a file over tramp, and then do `fasd-find-file`, I get

`Directory or file ‘/bin/bash: fasd: command not found’ doesn’t exist`

**Comment by steckerhalter** [comment]
Do you have fasd installed on the remote machine?

**Comment by unhammer** [comment]
No. I was hoping to get results from fasd run on my local machine …

**Comment by steckerhalter** [comment]
fasd doesn't work with remote file system. Maybe if you install fasd on the remote system it could work via tramp though.
